libs/community/langchain_community/chat_models/hunyuan.py

```diff
@@ -112,10 +117,10 @@ def lc_serializable(self) -> bool:
     """What sampling temperature to use."""
     top_p: float = 1.0
     """What probability mass to use."""
-    model: str = "hunyuan-lite"
+    model: str = "hunyuan-pro"
```

Comment:
this will quietly change the model type for anyone relying on the default model, here. Suggest keeping unchanged.

Reply:
ok，Restore Default Model